modules/op-running-unpriveleged-builds-in-a-container.adoc

```diff
@@ -0,0 +1,45 @@
+[id='op-running-unpriveleged-builds-in-a-container._{context}']
+= Running unprivileged builds in a container
+
+You can run builds as a non-root user inside a container. To run as a non-root user inside the container, you need your container to do some set up to add the `subuid` and `subgid` with the latest shadow-utils.
+
+The public image based on Red Hat UBI has everything already set up for us in its https://catalog.redhat.com/software/containers/ubi8/buildah/602686f7b16b1eb2e30807ee?container-tabs=dockerfile[Containerfile]. As an extra step, you need to “force” running the container as the “build” user since if we leave it to the default, it will be “randomed” and would not have the setup needed to run as non-root.
+
+You can directly edit or export to a specific namespace the buildah ClusterTask as described in the previous section. You can add this `securityContext`` to each https://github.com/tektoncd/operator/blob/main/cmd/openshift/operator/kodata/tekton-addon/addons/02-clustertasks/buildah/buildah-task.yaml#L63[build] and https://github.com/tektoncd/operator/blob/main/cmd/openshift/operator/kodata/tekton-addon/addons/02-clustertasks/buildah/buildah-task.yaml#L75[push] steps.
```

Comment:
@sferich888 or @RickJWagner please review these links to github.com. If needed, would you suggest alternatives?

Reply:
no github links (.)
there is no reason why our OCP docs' can't mirror down the content of those links (or capture what it is trying to say/show in our product docs).

Reply:
Please remove these links to github.com. If we need the content, Red Hat should publish similar information so that we can link to it. 
To summarize Eric's PoV:
"No github links (.)
Docs needs to enforce this.
There is no reason why our OCP docs' can't mirror down the content of those links (or capture what it is trying to say/show in our product docs)."